Dockerfile.ci

```diff
@@ -2,7 +2,7 @@ FROM golang:1.21-bullseye
 
 ARG APP_PATH
 
-RUN go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@v0.7.0
+RUN go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@v0.19.4
```

Comment:
Update the version to latest to get the docs generated correctly.

I've attempted several methods to resolve this error [https://github.com/okta/terraform-provider-oktapam/actions/runs/11475253141/job/31932723253?pr=125](vscode-file://vscode-app/Applications/Cursor.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html), but haven't had any success. Any thoughts?

Reply:
fixed in https://github.com/okta/terraform-provider-oktapam/pull/125/commits/50ae212fc747868c2e5f18c57d3ace79b39027dd